SYCL/ESIMD/regression/double_conversion_test.cpp

```diff
@@ -0,0 +1,82 @@
+//==--- double_conversion_test.cpp - DPC++ ESIMD on-device test ---------==//
```

Comment:
Generally speaking, the suffix "_test" is redundant because almost all of those files in 'regression' filder are tests.